package.json

```diff
@@ -23,8 +23,9 @@
     "turbo": "latest"
   },
   "engines": {
-    "npm": ">=7.0.0",
-    "node": ">=14.0.0"
+    "npm": "8.x",
+    "node": "16.x",
+    "yarn": "1.x"
```

Comment:
this is required for Heroku backend server to know how to setup our runtime environment